src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs

```diff
@@ -638,6 +642,8 @@ internal static List<TelemetryEvent> ToTelemetryPackageList(List<Tuple<string, s
 
             actionTelemetryEvent["TopLevelVulnerablePackagesCount"] = topLevelVulnerablePackagesCount;
             actionTelemetryEvent.ComplexData["TopLevelVulnerablePackagesMaxSeverities"] = topLevelVulnerablePackagesMaxSeverities;
+            actionTelemetryEvent["TransitiveVulnerablePackagesCount"] = transitiveVulnerablePackagesCount;
+            actionTelemetryEvent.ComplexData["TransitiveVulnerablePackagesMaxSeverities"] = transitiveVulnerablePackagesMaxSeverities;
```

Comment:
MaxSeverities being a list is...weird :D 
Max makes me think 1. I'm not sure what I'd be doing with a list anyways.

Reply:
Good observation. The history behind this is that some operations allow a list of packages. One such operation is Update. Install does not. The telemetry is designed to put a count of vulnerable packages in the operation plus a list of the max severity per vulnerable package that was installed. The count of the list of max severities should equal the count of vulnerable packages in the operation.

Reply:
So it's the max severity of each vulnerable package individually? 

I hate naming. 🤦